Code/Framework/AzCore/Tests/DOM/DomPrefixTreeBenchmarks.cpp

```diff
@@ -59,7 +59,7 @@ namespace AZ::Dom::Benchmark
     BENCHMARK_DEFINE_F(DomPrefixTreeBenchmark, FindValue_ExactPath)(benchmark::State& state)
     {
         SetupTree(state);
-        for (auto _ : state)
+        for ([[maybe_unused]] auto _ : state)
```

Comment:
I am assuming you are using VS2022 and these were flagged as warnings?

Because this change is unrelated to the AZ::Environment work, though I am OK with submitting these changes as a rider bill

Reply:
No, I built with Clang on Windows because I like its diagnostic messages better (and to prevent constant switching between Windows and Linux for the initial development).